app/error_handlers.py

```diff
@@ -56,7 +56,9 @@ def create_error_middleware(overrides):
 
 
 async def response_error(request, status: int):
-    flash(request, f"{status} " + SERVER_ERROR_MSG)
+    status_error = SERVER_ERROR_MSG.copy()
+    status_error['text'] = f"{status} " + status_error.get('text', '')
```

Comment:
I think we can remove the status code from here and keep the cause of the error vague for the respondent.